game.go

```diff
@@ -1271,46 +1271,66 @@ func (p *Game) loadSound(name string) (media Sound, err error) {
 
 // Play func:
 //
-//	Play(sound)
+//	Play(sound) -- sync
+//	Play(soundName) -- sync
+//	Play(sound, options) -- sync
+//	Play(soundName, options) -- sync
+//	StartPlay(sound) -- async
+//	StartPlay(soundName) -- async
+//	StartPlay(sound, options) -- async
+//	StartPlay(soundName, options) -- async
 //	Play(video) -- maybe
-//	Play(media, wait) -- sync
-//	Play(media, opts)
-//	Play(mediaName)
-//	Play(mediaName, wait) -- sync
-//	Play(mediaName, opts)
-func (p *Game) Play__0(media Sound) {
-	p.Play__2(media, &PlayOptions{})
+func (p *Game) Play__0(sound Sound, options *PlayOptions) {
+	p.play(sound, options, true)
 }
 
-func (p *Game) Play__1(media Sound, wait bool) {
-	p.Play__2(media, &PlayOptions{Wait: wait})
+func (p *Game) Play__1(soundName string, options *PlayOptions) {
```

Comment:
We need to restore the original order of overloads after the overload issue fixed by gop.